thunder/__init__.py

```diff
@@ -199,9 +199,11 @@
 apex_executor: None | extend.Executor = extend.get_executor("apex")
 nvfuser_executor: None | extend.Executor = extend.get_executor("nvfuser")
 pytorch_executor: None | extend.Executor = extend.get_executor("torch")
+custom_op_executor: None | extend.Executor = extend.get_executor("custom_op")
 
-# Default executor list is [cudnn -> sdpa -> torchcompile_cat -> torchcompile_xentropy -> nvfuser -> torch -> python]
+# Default executor list is [cudnn -> sdpa -> torchcompile_cat -> torchcompile_xentropy -> nvfuser -> custom_op -> torch -> python]
 # Note that add_default_executor inserts executor at start of list, hence the reverse order below.
+add_default_executor(custom_op_executor)
```

Comment:
Given that `custom_op_executor` has type `None | extend.Executor`, should there be a guard here?

```suggestion
if custom_op_executor:
    add_default_executor(custom_op_executor)
```